client/src/index.css

```diff
@@ -13,13 +13,24 @@
   src: url(./fonts/BarlowSemiCondensed-Black.ttf) format('truetype');
   font-weight: 900
 }
-
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900|Signika:300,400,600,700&display=swap" rel="stylesheet">
```

Comment:
Should this be in the html?

Reply:
The best way to include files in a react project would be to host them on out server. This has to do with how webpack works. See this: https://stackoverflow.com/questions/41676054/how-to-add-fonts-to-create-react-app-based-projects